api.md

```diff
@@ -1874,6 +1877,83 @@ are meant to implement data retention or perform tasks that will improve query
 performance on older chunks. Each policy is assigned a scheduled job
 which will be run in the background to enforce it.
 
+## add_job() :community_function: [](add_job)
+
+Register a custom procedure or function to be scheduled by our automation framework.
+
+#### Required Arguments [](add_job-required-arguments)
+
+|Name|Description|
+|---|---|
+| `proc` | (REGPROC) Name of the function or procedure to register as job|
+| `schedule_interval` | (INTERVAL) Interval between executions of this job|
+
+#### Optional Arguments [](add_job-optional-arguments)
+
+|Name|Description|
+|---|---|
+| `config` | (JSONB) Job-specific configuration (this will be passed to the function when executed)|
+| `initial_start` | (TIMESTAMPTZ) Time of first execution of job. |
+| `scheduled` | (BOOLEAN) Set to `FALSE` to exclude this job from scheduling. Defaults to `TRUE`. |
+
+#### Returns [](add_job-returns)
+
+|Column|Description|
+|---|---|
+|`job_id`| (INTEGER)  TimescaleDB background job id |
+
+
+#### Sample Usage [](add_job-examples)
+
+
+```sql
+SELECT add_job('custom_proc','1h', config:='{"type":"procedure"}'::jsonb);
```

Comment:
It would be more helpful if we first show the create procedure/function stmt for a function that takes an argument. And then show how to add  a job for that procedure/function as the next step. 

Also need to mention what is a valid config and the minimum information for the jsonb config. Can it be NULL for instance? Or do we always have to pass in type information?

Reply:
Agree.  I think here we will largely point to the "using custom jobs" section of the docs which will go into depth (forthcoming), but it would be nice to have a very single example here where you just show very simple definition of the UDF, then use that example to `add_job`.